akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -50,6 +52,16 @@ class Http2ClientSpec extends AkkaSpecWithMaterializer("""
   "The Http/2 client implementation" should {
     "support simple round-trips" should {
       abstract class SimpleRequestResponseRoundtripSetup extends TestSetup with NetProbes {
+
+        val defaultExpectedHeaders = Seq(
+          ":method" -> "GET",
+          ":scheme" -> "https",
+          ":authority" -> "www.example.com",
+          ":path" -> "/",
+          "content-length" -> "0",
+          "user-agent" -> ClientConnectionSettings(system).userAgentHeader.get.value
+        )
```

Comment:
I've extracted this block (copy/pasted multiple times) to add the `User-Agent` header (which requires a system to load the settings).